code/modules/clothing/head/tinfoilhat.dm

```diff
@@ -67,3 +71,24 @@
 	. = ..()
 	if(!warped)
 		warp_up()
+
+/obj/item/clothing/head/foilhat/proc/call_suicide(datum/source)
+	SIGNAL_HANDLER
+	INVOKE_ASYNC(src, /obj/item/clothing/head/foilhat.proc/suicide_act, source)	//SIGNAL_HANDLER doesn't like to wait, so i decided to move it to a proc in INVOKE_ASYNC
```

Comment:
```suggestion
	INVOKE_ASYNC(src, .proc/suicide_act, source) //SIGNAL_HANDLER doesn't like to wait, so i decided to move it to a proc in INVOKE_ASYNC
```

No inline tabs.

The comment isn't super useful regardless since this behavior is common, but since it's a "why" comment I'll allow it